lib/davinci_dtr_test_kit/payer_server_groups/adaptive_form_libraries_test.rb

```diff
@@ -9,6 +9,9 @@ class PayerAdaptiveFormLibrariesTest < Inferno::Test
       Inferno check that the payer response contains no duplicate library names
        and that libraries contain cql and elm data.
     )
+    verifies_requirements 'hl7.fhir.us.davinci-dtr_2.0.1@95', 'hl7.fhir.us.davinci-dtr_2.0.1@96',
```

Comment:
Requirement 84 is checked by the check_libraries method - see cql_test.rb line 257. Add to this test and similar ones

Reply:
Addressed in [6653e49](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/59/commits/6653e495e069efcc7e8474e6becffb5493393339)